packages/uikit/src/components/dashboard/DashboardTable.tsx

```diff
@@ -100,8 +101,10 @@ const isNumericColumn = (columnType: DashboardColumnType): boolean => {
 export const DashboardTable: FC<{ className?: string }> = ({ className }) => {
     const { data: columns } = useDashboardColumnsAsForm();
     const { data: dashboardData } = useDashboardData();
-    const { account } = useAppContext();
-    const publicKeys = account?.publicKeys;
+    const { data: wallets, isFetched: isWalletsFetched } = useWalletsState();
+    const mainnetPubkeys = wallets
+        ?.filter(w => w?.network === Network.MAINNET)
```

Comment:
if `w?.network === undefined` it's also mainnet

to get list of all mainnet wallet, please user `w?.network !== Network.TESTNET`